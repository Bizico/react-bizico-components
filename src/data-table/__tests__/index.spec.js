import React from 'react';
import {DataTable} from '../';
import renderer from 'react-test-renderer';

describe('DataTable', () => {

  it("should create component", function() {

		const component = renderer.create(
			<DataTable />
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

});
