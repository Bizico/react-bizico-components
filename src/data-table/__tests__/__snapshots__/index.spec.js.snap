exports[`DataTable should create component 1`] = `
<div>
  <table
    className="table" />
  <div
    className="red">
    test
  </div>
</div>
`;
